packages/account/src/Components/poi/poi-form-on-signup/idv-doc-submit-on-signup/idv-doc-submit-on-signup.tsx

```diff
@@ -129,8 +129,11 @@ export const IdvDocSubmitOnSignup = ({
         }
 
         if (!document_number) {
+            const document_name = document_type.text.toLowerCase();
```

Comment:
Can the `document_type.text` be undefined?

Reply:
let me include ? 👌

Reply:
I think it's better to check if it is defined in line 131